draft SIG (#165)
canton-foundation/canton-dev-fund: sig-review-process.md

# Development Fund Proposal Review
## Ways of Working

To manage the growing number of Development Fund proposals, the community will use a structured review workflow coordinated through the proposal board:

[https://github.com/orgs/canton-foundation/projects/3/views/1](https://github.com/orgs/canton-foundation/projects/3/views/1)

This process relies on participation from SIG members, Core Contributors, the Security group, and the Voting Committee.

---

# 1. Proposal Assignment

Proposal authors should indicate which **Special Interest Group (SIG)** they believe their proposal best aligns with when submitting their proposal. This helps reviewers quickly identify proposals that match their expertise.

> Members of the various SIGs are encouraged to review proposals aligned with their area of interest and may **self assign proposals they want to review directly on the proposal board.**

Reviewer: ```suggestion
Members of the various SIGs are encouraged to review proposals aligned with their area of interest and may use the proposal tracking board to **self assign proposals they want to review directly.**
```

Special Interest Groups are open to any ecosystem participant, including:

- Application developers  
- Node operators  
- Foundation members  

> Participation in SIGs is **not limited to Foundation members**.

Reviewer: ```suggestion
Currently, Special Interest Groups are open to any ecosystem participant, including:

- Application developers  
- Node operators  
- Foundation members  

Participation in SIGs is **not limited to Foundation members**.

The Technology & Operations committee may review this policy and limit SIG participation if required to meet the goals of the committee. 
```

When reviewing proposals:

> - Having an assignee will move the proposal to `In Review`

Reviewer: ```suggestion
- A proposal moves to 'In Review' once it has been assigned to a champion or reviewer. 
```
